<launch>
  <arg name="camera_id" default="0"/>
  <arg name="camera_name" default="left"/>
  <arg name="camera_reso" default="720"/>

  <arg name="camera_info_url" default="file://$(find usbcam)/../../../data/eye/left/ost.yaml"/>

  <node pkg="usbcam" type="usbcam_node" output="screen" name="$(arg camera_name)" >
    <param name="camera_id" type="int" value="$(arg camera_id)"/>
    <param name="camera_name" type="str" value="$(arg camera_name)"/>
    <param name="camera_reso" type="int" value="$(arg camera_reso)"/>
    <param name="camera_info_url" type="str" value="$(arg camera_info_url)"/>
  </node>

  <node pkg="image_proc" type="image_proc" name="$(arg camera_name)_image_proc" output="screen" ns="$(arg camera_name)">
    <remap from="image" to="image_raw" />
  </node>
</launch>
